### PR TITLE
Start setting up election page

### PR DESCRIPTION
**Pull request by Remi-Gau**
This is a first pass. Needs improvement.

- [ ] improve text on election section in landing page (can be done from the `config.yml` file)
- [ ] improve content and add links in election page (`/content/elections/index.md`)

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -144,64 +146,84 @@ params:
...
+      - title: Open publishing
+        text: Papers shouldn't hide behind paywalls. We want to help create solutions to enable new, open-acess platforms and to facilitate free information.
+        icon: doc-sync
+
+  elections:
+    title: Join the OSSIG
+    subtitle: We are excited to announce this year's Nominations for OHBM 2021!
+    id: openumbrella
+    features:
+      - title: Insert here motivational text why you should join the OSSIG
```
Comment (Remi-Gau) [inline config.yaml:204]: needs updating

File: config.yaml [hunk 2]
```diff
@@ -144,64 +146,84 @@ params:
...
+        buttonLink: /elections
+        buttonText: More information
+        icon: laptop-cloud
+      - title: Please self-nominate!
+        text: Insert here motivational text why you should join the OSSIG
```
Comment (Remi-Gau) [inline config.yaml:209]: needs updating

File: config.yaml [hunk 3]
```diff
@@ -144,64 +146,84 @@ params:
...
+
+  elections:
+    title: Join the OSSIG
+    subtitle: We are excited to announce this year's Nominations for OHBM 2021!
+    id: openumbrella
+    features:
+      - title: Insert here motivational text why you should join the OSSIG
+        buttonLink: /elections
+        buttonText: More information
+        icon: laptop-cloud
```
Comment (Remi-Gau) [inline config.yaml:207]: need to change the images for this section

File: content/elections/index.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+Take the lead on coordinating the Hackathon for next year! Apply in pairs of two. Work with the rest of the OS SIG members to put on the best Hackathon happening anywhere.
+
+- Number of positions: 2
+- Duration: 1 year
+
+### Inclusivity Officer: 
+
+Work together with the OS SIG and OHBM community to maximize inclusivity and eliminate exclusionary barriers in open-science and OHBM.
+
+- Number of positions: ???
```
Comment (Remi-Gau) [inline content/elections/index.md]: duration of the inclusivity officer

File: content/elections/index.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+We are excited to announce this year's Nominations for the OHBM open-science Special Interest Group (OS SIG) for OHBM 2021!
+
+Please self-nominate yourself [HERE](https://forms.office.com/Pages/ResponsePage.aspx?id=DQSIkWdsW0yxEjajBLZtrQAAAAAAAAAAAAMAAC9pqdJUN0xITDMxUktDT0RCUjJMVlc1OTVSRTlTRS4u!)
+
+The open-science movement is one of the best things happening in neuroscience. Hackathons, open-science room, new tools and methods, reproducibility and reliability, open access and inclusivity: the OS SIG is the place where it’s all happening!
+
+If open-science excites you and you want to help push its frontiers, please consider applying for a position in the OS SIG! 
+
+Early career researchers (Link to SIG), BIPOC, LGBTQ+ (Link to SIG), and people from outside Europe and North America especially encouraged to apply!
```
Comment (Remi-Gau) [inline content/elections/index.md:21]: Missing links

File: config.yaml (hunk 1 above)
Comment (62442katieb) [reply config.yaml:204]: Hi @Remi-Gau! Did you want me to suggest text for things like `Insert here motivational text why you should join the OSSIG`? I can do it in suggestions or my own PR, which would you prefer?

Comment (Remi-Gau) [reply config.yaml:204]: I am not fussy. Whichever is more convenient for you. :-)

File: content/elections/index.md (hunk 4 above)
Comment (62442katieb) [reply content/elections/index.md]: Inclusivity officer: 1 person, for 1 year

Comment (Remi-Gau) [inline content/elections/index.md]:
```suggestion
- Number of positions: 1
```

File: content/elections/index.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+- Number of positions: 2
+- Duration: 1 year
+
+### Inclusivity Officer: 
+
+Work together with the OS SIG and OHBM community to maximize inclusivity and eliminate exclusionary barriers in open-science and OHBM.
+
+- Number of positions: ???
+- Duration: ??? year
```
Comment (Remi-Gau) [inline content/elections/index.md]:
```suggestion
- Duration: 1 year
```

File: config.yaml (hunk 1 above)
Comment (62442katieb) [reply config.yaml:204]: I'll just reply here, then!

File: content/elections/index.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+date: 2020-06-30
+draft: false
+sidebar: false # true or false to display the sidebar
+sidebarlogo: fresh-white-alt # From (static/images/logo/)
+include_footer: true
+---
+
+# Make a difference: join the OS SIG!
+
+We are excited to announce this year's Nominations for the OHBM open-science Special Interest Group (OS SIG) for OHBM 2021!
```
Comment (62442katieb) [inline content/elections/index.md:13]: We've chatted a bit on the Mattermost channel about differentiating between the OSSIG (the whole community) and the OSSIG committee/officers (those official leadership positions). If we want to move forward with that, I might add "committee" or "officers" after `(OS SIG)` here.

File: content/elections/index.md (hunk 5 above)
Comment (62442katieb) [reply content/elections/index.md:21]: Student & postdoc sig: https://www.ohbmtrainees.com/
Diversity & Inclusion is a committee and doesn't have a website? But the committees are linked here: https://www.humanbrainmapping.org/i4a/pages/index.cfm?pageID=3313&activateFull=true

Are those the links you were looking for?

File: content/elections/index.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
+---
+title: "OHBM OSSIR 2021 elections"
```
Comment (62442katieb) [inline content/elections/index.md:2]: "elections" is tricky verbiage as we're not putting it to a popular vote this year 😬. Maybe swap "elections" for "committee" or "officers"? I think the student & postdoc SIG still refers to the process of selecting new officers as "elections", though, and doesn't actually vote either. Just a thought.

File: config.yaml (hunk 1 above)
Comment (62442katieb) [reply config.yaml:204]: `Insert here motivational text why you should join the OSSIG`:

If you are excited by open science, have enjoyed and benefitted from Hackathon and/or Open Science Room activity in your local hubs and globally, or have ideas about how we can improve on OHBM's open science activities in the future, please consider joining us! By self-nominating for a role as an Open Science SIG officer, you can join our passionate and enthusiastic team and help push for more reproducibility, transparency, and openness in science, as well as inclusivity and openness in our community. Early career researchers, BIPOC, LGBTQ+, and people from outside North America and Europe are especially encouraged to apply!

(any edits are 100% welcome)

File: content/elections/index.md (hunk 7 above)
Comment (cmaumet) [reply content/elections/index.md:13]: I like that! I think this confusion has been going on for too long ;)

File: content/elections/index.md (hunk 8 above)
Comment (cmaumet) [reply content/elections/index.md:2]: @62442katieb: Good point! I think this is very much up to you to decide. Let us know what you prefer.

File: config.yaml (hunk 1 above)
Comment (62442katieb) [reply config.yaml:204]: > By self-nominating for a role as an Open Science SIG officer, you can join our passionate and enthusiastic team and help push for more reproducibility, transparency, and openness in science, as well as inclusivity and openness in our community.

Edit from Cass: By self-nominating for a role as an Open Science SIG officer, you can join our passionate and enthusiastic team and help push for more reproducibility, transparency, and openness in science, as well as *improved inclusivity and access in our community.*

File: config.yaml (hunk 2 above)
Comment (62442katieb) [reply config.yaml:209]: If you are excited by open science, have enjoyed and benefitted from Hackathon and/or Open Science Room activity in your local hubs and globally, or have ideas about how we can improve on OHBM's open science activities in the future, please consider joining us! By self-nominating for a role as an Open Science SIG officer, you can join our passionate and enthusiastic team and help push for more reproducibility, transparency, and openness in science, as well as improved inclusivity and access in our community. Early career researchers, BIPOC, LGBTQ+, and people from outside North America and Europe are especially encouraged to apply!

**Pull request merged**